libnmstate/nm/wired.py

```diff
@@ -66,7 +66,8 @@ def create_setting(iface_state, base_con_profile):
     if speed:
         wired_setting.props.speed = speed
 
-    if duplex:
+    if duplex != 'unknown':
```

Comment:
This is not correct. This might be better:
```suggestion
    if duplex and duplex != 'unknown':
```

However, I guess the problem is what to do if auto-negotiation is reported as `false` but the device does not report a speed and duplex setting. Also I guess line 56 should be changed to not set `duplex` to `'unknown'` instead of checking it here.